spec/src/main/asciidoc/core/scopescontexts.asciidoc

```diff
@@ -187,7 +187,6 @@ The `@Dependent` scope is always active.
 
 Many instances of beans with scope `@Dependent` belong to some other bean and are called _dependent objects_.
 
-* Instances of decorators and interceptors are dependent objects of the bean instance they decorate.
```

Comment:
Should this line remain here, but refer only to interceptors?

(also the corresponding line in scopescontexts_full.asciidoc)

Reply:
Yeah, this should still be in place, good catch!